utils.py

```diff
@@ -17,3 +19,51 @@ async def fetch_plugin_json(url):
         response = await client.get(url)
         return response.json()
 
+
+def check_version_zip(name_plugin: str, version: int) -> bool:
+    """
+    Check if the cached plugin is updated.
+    
+    Returns:
+        False if is not updated or not exist (will be updated) otherwise return True
+    """
+    # Define a cache directory
+    cache_json = "zip_cache/" + "versions.json"
+    if not os.path.exists(cache_json):
+        with open(cache_json, "w") as json_file:
+                data = []
+                json.dump(data,json_file)
+                json_file.close() #idk why but if you don't have it, the code crash...
```

Comment:
when you use `with`, there is no need to call `.close()`. Resource is closed automatically ;)
I guess the problem is the double indentation

Reply:
maybe it is, but with the solution above, should be elimitated!